lib/active_storage_validations/aspect_ratio_validator.rb

```diff
@@ -5,13 +5,15 @@
 module ActiveStorageValidations
   class AspectRatioValidator < ActiveModel::EachValidator # :nodoc
     include OptionProcUnfolding
+    include ErrorHandler
 
     AVAILABLE_CHECKS = %i[with].freeze
     PRECISION = 3
 
     def check_validity!
-      return true if AVAILABLE_CHECKS.any? { |argument| options.key?(argument) }
-      raise ArgumentError, 'You must pass :with to the validator'
+      unless AVAILABLE_CHECKS.any? { |argument| options.key?(argument) }
+        raise ArgumentError, 'You must pass :with to the validator'
+      end

```

Comment:
Method does not return `true` on success anymore. Is that intended?

Reply:
Sorry for the late answer, I did not see your comment!
For this change, I mimicked Rails implementation of this method ([https://api.rubyonrails.org/classes/ActiveModel/EachValidator.html#method-i-check_validity-21](https://api.rubyonrails.org/classes/ActiveModel/EachValidator.html#method-i-check_validity-21))

It looks like that returning true is not a requirement, this method should only check if the arguments are valid raising an error if not valid. Therefore I took the liberty of removing it, let me know if I overstepped on that one!

Reply:
@gr8bit Should I change something on the PR? It's been running for quite some time already :'(

Reply:
probably this is all good, any result except ArgumentError is fine